crates/runtime/src/gc/enabled.rs

```diff
@@ -283,14 +286,12 @@ impl VMExternData {
 
     #[inline]
     fn increment_ref_count(&self) {
-        // This is only using during cloning operations, and like the standard
-        // library we use `Relaxed` here. The rationale is better documented in
-        // libstd's implementation of `Arc`, but the general gist is that we're
-        // creating a new pointer for our own thread, so there's no need to have
-        // any synchronization with orderings. The synchronization with other
-        // threads with respect to orderings happens when the pointer is sent to
-        // another thread.
-        self.ref_count.fetch_add(1, Ordering::Relaxed);
+        self.ref_count.set(self.ref_count.get() + 1);
+        log::trace!(
+            "Incrementing ref count of externref @ 0x{:p} -> {}",
+            self as *const _,
```

Comment:
Could this debug print be `self.0.as_ptr()` since `&self` changes enough that its pointer value probably isn't too interesting?

Reply:
This is a method on `VMExternData`, not `VMExternRef`, so `&self` is the interesting pointer value.